### PR TITLE
API Updates for beta branch

### PR DESCRIPTION
**Pull request by pakrym-stripe**
Codegen for openapi v184.
r? @dcr-stripe
cc @stripe/api-libraries

## Changelog
- Updated stable APIs to the latest version

### CODE REVIEW

File: stripe/api_resources/plan.py [hunk 1]
```diff
@@ -13,4 +13,15 @@ class Plan(
     ListableAPIResource,
     UpdateableAPIResource,
 ):
+    """
+    You can now model subscriptions more flexibly using the [Prices API](https://stripe.com/docs/api#prices). It replaces the Plans API and is backwards compatible to simplify your migration.
+
+    Plans define the base price, currency, and billing cycle for recurring purchases of products.
+    [Products](https://stripe.com/docs/api#products) help you track inventory or provisioning, and plans help you track pricing. Different physical goods or levels of service should be represented by products, and pricing options should be represented by plans. This approach lets you change prices without having to change your provisioning scheme.
+
+    For example, you might have a single "gold" product that has plans for $10/month, $100/year, €9/month, and €90/year.
```
Comment (dcr-stripe) [inline stripe/api_resources/plan.py:22]: The euro symbols in this and price.py need to be manually changed to EUR: https://github.com/stripe/stripe-python/pull/861/commits/da09005a12e9b1483328a3de47d4aea090a8a4ec

Kamil is fixing this in the codegen layer but it's not in yet.

**Review by dcr-stripe — CHANGES_REQUESTED** [review]
Just one thing to change and then LG!

File: stripe/api_resources/plan.py (hunk 1 above)
Comment (pakrym-stripe) [reply stripe/api_resources/plan.py:22]: I'll regen when his fix is in.

Comment (pakrym-stripe) [inline stripe/api_resources/plan.py:22]:
```suggestion
    For example, you might have a single "gold" product that has plans for $10/month, $100/year, EUR 9/month, and EUR 90/year.
```

**Comment by pakrym-stripe** [comment]
r? @dcr

Updated.

**Pull request merged**